### ISSUE TITLE
Issues related to OpenAlex & ResearchHub Data Integrations

### ISSUE THREAD

**Issue by dominikusbrian** [ResearchHub/issues#96]
This Bug report is a collection of instances where the Data Integration/representation between OpenAlex and ResearchHub seems to not working properly or as intended. 

On 2nd August, not long after the V2 upgrade with OpenAlex integration included went live, I was running some query check and test, with a paper I published on ResearchHub (using the 5 RSC DOI option along with it). I search for the paper on OpenAlex, and got it assigned to other author (there is info about me but doesn't appear properly). Moreover there's issue with the Open Access status for paper published on ResearchHub within the OpenAlex system. This paper that is published on ResearchHub is categorized as Closed access for some reason. 

Perhaps now that OpenAlex is an integral part of ResearchHub larger infrastructure, it would be essential to iron through this details, and get a mid-ground solution to synchronise data in some way.

![image](https://github.com/user-attachments/assets/76112f35-c778-454a-ad8a-cde4ae2f78bc)

Meanwhile, in Semantics Scholar (https://semanticsscholar.org/) all the research article published on ResearchHub with DOI are all properly indexed and available for cite directly.

![image](https://github.com/user-attachments/assets/0dff855c-2f5b-4698-8fe4-a73c58a04b62)

That being said, interestingly, the citations of these articles published on ResearchHub, is designated as inProceeding and given a new url that point only to Semantics Scholar. Only after user arrived at Semantics Scholar entry of the paper, that user can take another step to find original link that points to ResearchHub.

We also notice, the automatically generated BibTex for this paper published on ResearchHub , not showing any info related to ResearchHub as publisher, platform, nor source.
![image](https://github.com/user-attachments/assets/84390b92-881c-421e-977d-ab7fc3696691)

Manually adding the publication by clicking the "+add publication" button on my ResearchHub Profile (Publication tab)  and putting in the ResearchHub DOI, doesn't give the correct author/data. So now even information that originally correct within the ResearchHub Dataset, now become wrong (because the OpenAlex database wrongly assigned the RH generated DOI)
![image](https://github.com/user-attachments/assets/f2ae6b3e-e585-4f74-ac41-a0ae264d01c5)

---

Follow up question on OpenAlex. I recently (2 days ago) published a paper that made it to OpenAlex within a day after it is online on the journal page. 

I checked my OpenAlex profile page which properly listed the new publication. But ResearchHub Profile, isn't updating this automatically even after 3 days. So I am wondering, what are the interval for API fetch or info update between RH and OpenAlex ?

**Comment by dominikusbrian** [comment]
Notably when there's error during the indexing of the ResearchHub generated content on OpenAlex system, the error would then to some extent propagate back to ResearchHub, everytime ResearchHub sync/pull latest data from OpenAlex.

**Comment by dominikusbrian** [comment]
Adding another instance of data mismatch need to be fixed. 
I found a duplicate of my profile here 

https://www.researchhub.com/author/1810994 [shown below in screenshot] 
This duplicate profile seems to be part of the automated author's profile import from OpenAlex.
Duplication occur, because prior to OpenAlex my account profile already exist and OpenAlex data import doesn't have way to check if the imported profile is actually related to any existing one. 

Not sure what's the right course of action here. Maybe worth thinking to avoid future misuse of this or creation of too many "Ghost Profiles" that may burden the server, unnecessarily. 

![image](https://github.com/user-attachments/assets/daf93a50-96f1-40c7-b835-852a98b74479)


For reference, my original and active author profile is here :
https://www.researchhub.com/author/993107 
![image](https://github.com/user-attachments/assets/1d533460-9352-4d45-95cf-397dacbd40f7)

**Comment by dominikusbrian** [comment]
Adding extra reported instances by the following author https://www.researchhub.com/author/1872994 , 
Details can be found well-described across the issue #100 #101 and #102

**Comment by dominikusbrian** [comment]
adding another instance of this error in author matching from ResearchHub posted DOI 

![image](https://github.com/user-attachments/assets/19d53422-db81-48d8-8d66-0dbe4926a61c)

**Comment by dominikusbrian** [comment]
Noticed a duplicate paper imported from OpenAlex. Interestingly the journal is labelled as ResearchHub Technologies, Inc. and license is marked as closed access. 

![image](https://github.com/user-attachments/assets/3a4e579e-435d-471b-b768-baa7cbbb9ce6)

This closed access entry is actually a duplicate. Searching for the paper title give two results:
one being Post ->  the original, and Paper  -> imported entry from OpenAlex Sync

![image](https://github.com/user-attachments/assets/ddab0926-1581-444b-9f63-e785c9525960)

**Comment by dominikusbrian** [comment]
Articles posted on ResearchHub with DOI, are indexed by OpenAlex as Non Open-Access. 

![image](https://github.com/user-attachments/assets/c321ceb3-a67b-46a5-9eff-ecea69d97765)
![image](https://github.com/user-attachments/assets/bc7d7c1b-b597-4b20-bba6-72746bc8bb5f)

This is the case for most (if not all) documents published natively on ResearchHub.
![image](https://github.com/user-attachments/assets/03060eb3-b9dd-4d7f-a66a-ac6693b17f67)
